src/pages/AboutPage/index.tsx

```diff
@@ -25,40 +33,72 @@ const CardsRow = ({ children }: { children?: ReactNode }) => (
   >
     {children}
   </Row>
-)
+);
+
+const SocialSection = ({ socials }: { socials?: Socials }) => {
+  if (!socials) return <></>;
+
+  return (
+    <>
+      <ColoredLine />
+      {socials.email && <p>Email: {socials.email}</p>}
```

Comment:
lets make some of these links clickable with an <a> tag

Reply:
Made links clickable in most recent change